Scripts_to_create_table/entities.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SEQUENCE entities_seq;
+CREATE TABLE entities(
+   entity_id integer PRIMARY KEY default nextval('entities_seq'),
+   name text NOT NULL,
+   pseudonym text[],
```

Comment:
Will you ever perform queries based on this, work or release? Storing data in arrays in a RDBMS is only wise if you use those fields as pure data-dumps. If you intend to use this for querying and the arrays or table gets big, this will introduce inefficiency.